# Generated by Django 2.2 on 2020-02-18 15:20

import datetime
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('mydiplom', '0008_auto_20200218_1618'),
    ]

    operations = [
        migrations.AddField(
            model_name='claim',
            name='finally_rejected_comment',
            field=models.TextField(default='Заявку невозможно обработать', max_length=10000, verbose_name='Причина отклонения'),
        ),
        migrations.AddField(
            model_name='claim',
            name='restore_request_text',
            field=models.TextField(default='Просьба восстановить заявку', max_length=10000, verbose_name='Причина отклонения'),
        ),
        migrations.AlterField(
            model_name='claim',
            name='first_rejected_comment',
            field=models.TextField(default='В данный момент все специалисты заняты', max_length=10000, verbose_name='Причина отклонения'),
        ),
        migrations.AlterField(
            model_name='myuser',
            name='birthday',
            field=models.DateField(default=datetime.datetime(1990, 2, 18, 5, 20, 53, 675370), verbose_name='Дата рождения'),
        ),
    ]
